### PR TITLE
Rest api tests

### PR DESCRIPTION
**Pull request by sc68cal**

### CODE REVIEW

File: kostyor/conf.ini [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [global]
 host = 127.0.0.1
 port = 5000
+discovery_methods = first,second
```
Comment (jvarlamova) [inline kostyor/conf.ini]: Dima, why did you add 'first' and 'second' discovery methods? Why didn't you add 'openstack'  method, which is already implemented in inventory/discover.py?

Comment (dstepanenko) [inline kostyor/conf.ini]: There are 2 types of discovery methods - default and non-default. Default
methods are always there and you can use them in every openstack setup for
every platform. Non-default methods are platform and setup-specific and
should be listed in the config file

29 июля 2016 г. 11:06 пользователь "Julia Varlamova" <
notifications@github.com> написал:

Comment (jvarlamova) [inline kostyor/conf.ini]: Thank you for explanation!

Comment (dstepanenko) [inline kostyor/conf.ini]: You're welcome :)

29 июля 2016 г. 11:25 пользователь "Julia Varlamova" <
notifications@github.com> написал:
